### DISCUSSION TITLE
The aggregate signature is invalid

### DISCUSSION THREAD

**Discussion by tangdasan** [Chia-Network/chia-blockchain#11253] [Support]
After upgrading the 1.3.x version, the node cannot obtain points, the same key, and everything is normal in version 1.2.11.

delete config, delete wallet db, tried everything i could,I have a 30P farm and it's bothering me,As a result, I can't upgrade all 1.3.x

Other keys are upgraded to 1.3.x, the same mining pool can get points, please don’t tell me to try switching pools!

please help me.

![image](https://user-images.githubusercontent.com/26919535/164442863-e8b5e479-94ea-43cc-9076-ef065f015e8b.png)

**Comment by scrutinously** [comment]
Error code 7 is a mismatch between the `authentication_public_key` in your config.yaml and what the pool has in their records. You need to contact the pool operator and provide them with your new `authentication_public_key` so they can update it on their end.

**Reply by tangdasan** [reply]
ok, I will try it, and I want to know if this problem occurs because of the same key, there are 1.3.x and 1.2.11?

**Reply by scrutinously** [reply]
I'm curious if you created the affected plotnft when pooling protocol was still in beta (version 1.1.7)? This was the case for several users on xchpool who joined during the "beta", and after they migrated their farm to a new system, generated a new config.yaml and resynced their wallet, it changed the `authentication_public_key` and required xchpool to update the key on their end.

**Reply by tangdasan** [reply]
I can't remember the date.

**Comment by emlowe** [comment]
Check your pool_list in config.yaml, I might consider making a backup of your config.yaml, removing the pool_list section and then resync your wallet which should put the proper pool_list items back into config.yaml after syncing.

**Reply by tangdasan** [reply]
Can you tell me which line to delete from?

**Reply by tangdasan** [reply]
![image](https://user-images.githubusercontent.com/26919535/164583833-5df6b1a7-80e8-4586-ad31-e8e21368b44c.png)

After I removed, plotnftshow shows this, I guess this is wrong

**Comment by tangdasan** [comment]
I think this must be some kind of bug, I will roll back 1.2.11 until more people raise this bug and let the developers notice

**Reply by tangdasan** [reply]
I noticed a lot of people in IM asking this question
